automated-tests/src/main/java/com/hp/ov/sdk/network/LogicalInterconnectGroupResource.java

```diff
@@ -135,6 +141,11 @@ public void createSynergy() {
     }
 
     @Override
+    public String patch(String id) {
+        return taskToString(client.patch(id, builderPatch(client.getById(id))));

```

Comment:
You don't have the patch method implemented for LIG in this branch. You need to get it (rebase) from master before using it. 
This branch has compilation errors and we cannot merge it. This is why you should do it changes all in the same branch.